### PR TITLE
Improve Metadata editor UI

### PR DESCRIPTION
**Pull request by LegoStormtroopr**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

**Comment by LegoStormtroopr** [comment]
See below:
image

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/generic/views.py [hunk 1]
```diff
@@ -536,6 +536,45 @@ class ExtraFormsetMixin:
...
+        logger.error(extra_formsets)
+        for item in extra_formsets:
+            logger.error(item)
+            if (
+                    item['formset'].non_form_errors() or (
+                        item['formset'].errors and
+                        not (len(item['formset'].errors) == 1 and item['formset'].errors[0] == {})
+                    )
+            ):
+                logger.critical(str([
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/generic/views.py]: Looks good, can we remove this critical log though

**Comment by harrywhite4** [comment]
Could maybe look at using these for the field errors instead of alerts https://getbootstrap.com/docs/3.3/css/#forms-control-validation

Those tab icons are super nice though

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/generic/views.py (hunk 1 above)
Comment (LegoStormtroopr) [reply python/aristotle-metadata-registry/aristotle_mdr/contrib/generic/views.py]: Can do

**Comment by LegoStormtroopr** [comment]
> Could maybe look at using these for the field errors instead of alerts https://getbootstrap.com/docs/3.3/css/#forms-control-validation

That would be much better - I had no idea they were a think.
We've pretty consistently used alerts wrongly, and changing to form errors will look nicer.
Can we do that as a separate PR?

**Comment by harrywhite4** [comment]
All good, will put that on the backlog

**Pull request merged**